ci/requirements/minimum-tests.yml

```diff
@@ -1,18 +1,25 @@
 name: climpred-minimum-tests
 channels:
   - conda-forge
-  - defaults
 dependencies:
-  - xesmf
-  - esmpy
```

Comment:
@andersy005, these have to stay up here. There's an extended conversation at https://github.com/JiaweiZhuang/xESMF/issues/47 about how `xesmf` breaks with `esmpy` and it's a really hard issue to resolve. The only sustainable solution we've found is to install `xesmf` prior to `esmpy`.  Although tests are running fine with this new order... but maybe it's something the tests don't catch but we get interactively in the dev environment. Any thoughts on this?

Reply:
How about specifying the `mpi` build for esmpy via

```yaml
- esmpy=*=mpi*
- xesmf
```
???

From reading comments from that issue, it's my understanding that the approach above would guarantee that conda/mamba will install `esmpy` built with `mpi` support.

Reply:
Yes that could work as well. I mean the current implementation seems to run the tests, and this is just the test environment and not dev environment so it's probably fine. So either go back to the original order with no mpi versioning, or implement this mpi versioning approach, or leave it as is. Let me know and I can merge once you decide what makes the most sense.